crypto/err/err.c

```diff
@@ -261,37 +279,36 @@ static void build_SYS_str_reasons(void)
     CRYPTO_THREAD_unlock(err_string_lock);
     /* openssl_strerror_r could change errno, but we want to preserve it */
     set_sys_error(saveerrno);
-    err_load_strings(SYS_str_reasons);
+    load_strings(SYS_str_reasons);
 }
 #endif
 
-#define err_clear_data(p, i) \
-        do { \
-            if ((p)->err_data_flags[i] & ERR_TXT_MALLOCED) {\
-                OPENSSL_free((p)->err_data[i]); \
-                (p)->err_data[i] = NULL; \
-            } \
-            (p)->err_data_flags[i] = 0; \
-        } while (0)
-
-#define err_clear(p, i) \
-        do { \
-            err_clear_data(p, i); \
-            (p)->err_flags[i] = 0; \
-            (p)->err_buffer[i] = 0; \
-            (p)->err_file[i] = NULL; \
-            (p)->err_line[i] = -1; \
-        } while (0)
-
-static void ERR_STATE_free(ERR_STATE *s)
+static void clear_data(ERR_ENTRY *p)
```

Comment:
You *could* make this `static ossl_inline`.  Entirely your choice, it's fine either way